src/joytabwidget.cpp

```diff
@@ -1297,6 +1299,17 @@ void JoyTabWidget::changeCurrentSet(int index)
         break;
     }
 
+
+    if(m_settings->value("Notifications/notify_about_set_change", false).toBool()){
+
+        ///This only works if this is a Linux system. This gives the possibility to add windows and mac notifications later on
+        #if defined(Q_OS_LINUX)
+
+            ///This takes the message to be echoed to the shell, inserts index (much cleaner than before), and sends it to the shell, triggering the notification
+            system(qPrintable(QString("notify-send \"AntiMicroX\" \"Set %1 is now active\" --urgency=normal -i \"/home/guttmann/Desktop/antimicrox/src/icons/antimicrox.ico\"").arg(index + 1)));
```

Comment:
` -i \"/home/guttmann/Desktop/antimicrox/src/icons/antimicrox.ico\"` - not everyone is guttmann ;)  
Use `"--icon=battery-caution"`

I would recommend replacing this line with something like this:
```cpp
QProcess process;
        process.start("notify-send", QStringList() << tr("Battery level is less than %1").arg(percent)
                                                   << tr("Device number: %1\nDevice name: %2")
                                                          .arg(device->getRealJoyNumber())
                                                          .arg(device->getSDLName())
                                                   << "--icon=battery-caution"
                                                   << "--urgency=normal");
```
And remember about putting strings for user in `tr()` - it will allow later translation to other languages